src/main/resources/locale_pt_BR.properties

```
@@ -0,0 +1,361 @@
+command.abuseprotection.cooldown.description=Obter ou definir o arrefecimento da reputação.
+command.abuseprotection.cooldown.message.get=Você pode enviar reputação para um usuário a cada %MINUTES% minutos.
+command.abuseprotection.cooldown.message.set=Cooldown definido para %MINUTES% minutos.
+command.abuseprotection.cooldown.minutes.description=Quantidade de minutos que os usuários não podem agradecer uns aos outros novamente.
+command.abuseprotection.description=Receba as configurações atuais de proteção contra abuso.
+command.abuseprotection.donorcontext.description=Obtenha ou defina se o doador precisa ou não de mensagens recentes.
+command.abuseprotection.donorcontext.message.false=O doador não precisa de mensagens recentes no canal.
+command.abuseprotection.donorcontext.message.true=O doador deve ter mensagens recentes no canal.
+command.abuseprotection.donorcontext.state.description=Defina como "verdadeiro" para ativar as verificações de contexto do doador.
```

I think the quoted text should be `true`?
```suggestion
command.abuseprotection.donorcontext.state.description=Defina como "true" para ativar as verificações de contexto do doador.
```